variables.tf

```diff
@@ -12,90 +36,96 @@ variable "description" {
+  description = "Set to true to enable the GitHub Wiki features on the repository. (Default: false)"
+  default     = null
 }
 
 variable "allow_merge_commit" {
   type        = bool
-  description = "Set to true to enable merge commits on the repository."
-  default     = false
+  description = "Set to true to enable merge commits on the repository. (Default: false)"
+  default     = null
 }
 
 variable "allow_squash_merge" {
   type        = bool
-  description = "Set to true to enable squash merges on the repository."
-  default     = false
+  description = "Set to true to enable squash merges on the repository. (Default: false)"
+  default     = null
 }
 
 variable "allow_rebase_merge" {
   type        = bool
-  description = "Set to true to enable rebase merges on the repository."
-  default     = false
+  description = "Set to true to enable rebase merges on the repository. (Default: false)"
+  default     = null
 }
 
 variable "has_downloads" {
   type        = bool
-  description = "Set to true to enable the (deprecated) downloads features on the repository."
-  default     = false
+  description = "Set to true to enable the (deprecated) downloads features on the repository. (Default: false)"
+  default     = null
 }
 
 variable "auto_init" {
   type        = bool
-  description = "Wether or not to produce an initial commit in the repository."
-  default     = true
+  description = "Wether or not to produce an initial commit in the repository. (Default: true)"
+  default     = null
 }
 
 variable "gitignore_template" {
   type        = string
   description = "Use the name of the template without the extension. For example, Haskell. Available templates: https://github.com/github/gitignore"
-  default     = ""
+  default     = null
 }
 
 variable "license_template" {
   type        = string
   description = "Use the name of the template without the extension. For example, 'mit' or 'mpl-2.0'. Available licences: https://github.com/github/choosealicense.com/tree/gh-pages/_licenses"
-  default     = ""
+  default     = null
 }
 
 variable "default_branch" {
   type        = string
   description = "The name of the default branch of the repository. NOTE: This can only be set after a repository has already been created, and after a correct reference has been created for the target branch inside the repository. This means a user will have to omit this parameter from the initial repository creation and create the target branch inside of the repository prior to setting this attribute."
-  default     = ""
+  default     = null
 }
 
 variable "archived" {
   type        = bool
-  description = "Specifies if the repository should be archived."
+  description = "Specifies if the repository should be archived. (Default: false)"
   default     = false
 }
 
 variable "topics" {
   type        = list(string)
-  description = "The list of topics of the repository."
+  description = "The list of topics of the repository.  (Default: [])"
```

Comment:
Do we really want to put the type into the description? The IDE should already show us the right type, no?

Reply:
The default is now null.. which is just the Terraform default but not the real default anymore as the module defines the default now to be more flexible..